Test/DependencyCollector/FunctionalTests/FuncTest/SqlTests.cs

```diff
@@ -23,6 +24,11 @@ public class SqlTests
         private const string ResourceNameSQLToDevApm = @".\SQLEXPRESS | RDDTestDatabase";
 
         /// <summary>
+        /// Resource Name for wrong database.
+        /// </summary>
+        private const string ResourceNameSQLToWrongDatabase = @"invalid\SQLEXPRESS | RDDTestDatabase";
```

Comment:
Have you tested auth errors in SQL.Open? DNS errors typically thrown before any attempt to open connection will be made so the flow of the code execution for different errors will be different